generators/app/templates/central-server-config/_application.yml

```diff
@@ -6,7 +6,7 @@ jhipster:
         authentication:
             jwt:
                 secret: <%= jwtSecretKey %>
-    <% if (useElk) {%>
+<%_ if (useElk) { _%>
```

Comment:
<%_ & _%> slurps the newline and hence we can use it for properly formatted
scripting without worrying about newlines in the output

Thanks & Regards,
Deepu

On Tue, Mar 15, 2016 at 9:28 PM, William Marques <notifications@github.com>
wrote:

> In generators/app/templates/central-server-config/_application.yml
> <https://github.com/jhipster/generator-jhipster-docker-compose/pull/7#discussion_r56161462>
> :
>
> > @@ -6,7 +6,7 @@ jhipster:
> >          authentication:
> >              jwt:
> >                  secret: <%= jwtSecretKey %>
> > -    <% if (useElk) {%>
> > +<%_ if (useElk) { _%>
>
> What's the difference between using <% and <%_ ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
>
> https://github.com/jhipster/generator-jhipster-docker-compose/pull/7/files#r56161462
>